source/includes/usage-examples/write-code-examples.java

```diff
@@ -0,0 +1,77 @@
+# start-insert-one
+Document document = new Document("<field name>", "<value>");
+Mono<InsertOneResult> result = Mono.from(collection.insertOne(document));
+
+System.out.printf("Inserted 1 document with ID %s%n.", 
```

Comment:
If you want to end a message with `.`, that should be done before printing a new line:

```suggestion
System.out.printf("Inserted 1 document with ID %s.%n", 
```

Sometimes there is also `%s%n documents.`, which should be `%s documents.%n`.

This comment is applicable to multiple places.